docker/test-e2e/Dockerfile

```diff
@@ -7,14 +7,12 @@ RUN npm config set unsafe-perm true && npm install -g npm@7.6.3
 RUN mkdir -p /data
 WORKDIR /data
 
-# unsafe-perm true is required to work around an npm bug since we are running as root, have a git+HTTPS repo source and it has a `prepare` script (perfect storm).
-# see https://github.com/npm/npm/issues/17346
-COPY package.json package-lock.json ./
-RUN npm config set unsafe-perm true && npm install --legacy-peer-deps
+COPY package.json ./
+COPY node_modules ./node_modules/
```

Comment:
Here the comment about `npm rebuild node-sass` does not apply, so it might be possible to volume-mount `node_modules` here and reduce image size. OTOH, this is an image we don't deploy, so if this image is large it doesn't cost us much. And since this image is going to go away eventually, paying a complexity cost by changing it like that doesn't make as much sense.

Reply:
Yes, at this point, this image is only built on developer machines and is neither built nor published on GHA.